spec/models/github_info_search_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe GithubInfoSearch do 
+
+  describe 'attributes' do 
+
+    before(:each) do
+      @token = ENV['GITHUB_USER1_TOKEN']
+      @search = GithubInfoSearch.new(@token)
+    end
+
+    it {expect(@search).to be_an_instance_of(GithubInfoSearch)} 
+    it {expect(@search).to have_attributes(token: @token)}
+
+  end 
+
+  describe 'methods' do
+
+    before(:each) do
+      @token = ENV['GITHUB_USER1_TOKEN']
+      @search = GithubInfoSearch.new(@token)
+    end
+
+    it "returns repositories" do
+      VCR.use_cassette('/github_info_search/returns_repositories') do
+        @search.repositories.each do |repository| 
+          expect(repository).to be_an_instance_of(Repository)
```

Comment:
I did not know you could use the `instance_of` outside of minitest 0.o Good work!